src/Refractor.js

```diff
@@ -33,7 +33,23 @@ function Refractor(props) {
 
   const value = ast.length === 0 ? props.value : ast.map(mapChildren.depth(0))
 
-  const code = h('code', codeProps, value)
+  const code = h(
+    'code',
+    codeProps,
+    props.lineNumbers
+      ? [
+          ...value,
+          h(
+            'span',
+            {className: 'line-numbers', 'aria-hidden': true, key: 'fract-line-numbers'},
+            props.value
+              .split('\n')
+              .slice(1)
```

Comment:
Why the slice? It seems it results in one missing line number at the end?

Reply:
I'm using `react-refractor` along with MDX, say I have this code in mdx:

<pre>
```bash
npm init voidjs ./my-app
```
</pre>

Here's the value passed to `react-refractor`:

![image](https://user-images.githubusercontent.com/1091472/110891489-0f655c00-832d-11eb-9df4-bdf35014994b.png)

Thus I sliced it.

But yeah, it might cause trouble. I've updated the code to remove it only empty line detected at the end.